lab-saul/lib/server.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const express = require('express');
+const mongoose = require('mongoose');
+
+mongoose.Promise = Promise;
+mongoose.connect(process.env.MONGODB_URI);
+
+let server;
+const app = express();
+
+//DONT NEED THIS????
+// app.get('/api/'hello.erkegdigfkepik)
+
+
+app.use(require('../route/routes-info-store.js'));
+app.use((err, req, res, next) => {
+  console.log('err', err);
+  if(!err){
+    res.sendStatus(500);
+  }
```

Comment:
We created some error middleware in class that would be helpful for getting that duplicate key error status that your looking for that's breaking your POST request test.